### ISSUE TITLE
TODO items still won't automatically update in [/] or [%] in heading

### ISSUE THREAD

**Issue by BigPeet** [jceb/vim-orgmode#244]
How was the issue #172 closed? It still doesn't work as it is currently documented. It only checks direct children with a check boxes, but it completely ignores TODO or DONE. This example taken from the documentation does NOT work:

* Organize Party [33%]
** TODO Call people [1/2]
*** TODO Peter
*** DONE Sarah
** TODO Buy food
** DONE Talk to neighbor

It won't update the % of "Organize Party" and it won't update the / of Call People unless you replace every TODO and DONE with check boxes.

I don't want to be rude or something, but I would love this to work and it just doesn't.

**Comment by schoettl** [comment]
I guess it's not yet implemented. It would be a very nice feature. It can also work for all keyword sets (not only `TODO` and `DONE`) because every keyword set has "open" and "closed" keyword(s). See `:help org_todo_keywords`.

It's probably quiet a bit of work. I think the issue should remain open. Maybe some day someone has time to implement this :)

**Comment by thriveth** [comment]
Does this mean it is *known* to not be implemented? Because the helpfile that comes with the plugin says it should work,. so at least this could be updated to properly reflect the actual state of the functionality? I'd rather read that it is not implemented than believe it is and then discover it's not.

**Comment by schoettl** [comment]
vim-orgmode's doc is taken from the original orgmode doc: http://orgmode.org/manual/Breaking-down-tasks.html

I think too, there should be a hint in the documentation. I created a pull request: https://github.com/jceb/vim-orgmode/pull/256

**Comment by jporsay** [comment]
I got it to work using lists:
```
* TODOs [33%]
  - [-] High
    - [X] high 1
    - [ ] high 2
  - [X] Med
    - [X] med 1
  - [ ] Low
    - [ ] low 1
```

**Comment by thriveth** [comment]
@jporsay  Yes, that worked all along, but these subtasks don't go into agenda items. So it's usable, but could be more useful.

**Comment by thriveth** [comment]
@jporsay  Yes, that worked all along, but these subtasks don't go into agenda items. So it's usable, but could be more useful.

**Comment by jporsay** [comment]
@thriveth Oh, you're right, didn't think about the agenda items.

**Comment by Swoorup** [comment]
came from todo.txt. Disappointed to see this not working. Everything else is great so far.

**Comment by NotAFile** [comment]
Just got bitten by this too. The "partially implemented" is a bit unclear IMO.

**Comment by mcepl** [comment]
I like the idea of vim-orgmode, but for anybody who would need this actually working, I can point you to the project I co-maintain, https://github.com/vimoutliner/vimoutliner, where this actually works:
```
[_] 8% bsc#1155094 CVE-2019-18348 CRLF injection
	[_] 33% python
		[x] SLE-10-SP3:Update
		[_] SLE-11-SP1:Update
		[_] SLE-12-SP1:Update sr#211064
	[_] 0% python27
		[_] SLE-11-SP1:Update:Teradata
	[_] 0% python36
		[_] SLE-12-SP3:Update:Products:Teradata:Update
	[_] 0% python3
		[_] SLE-12:Update
```
